packages/arb-token-bridge-ui/tests/e2e/specs/deposit.cy.ts

```diff
@@ -0,0 +1,98 @@
+import { JsonRpcProvider } from '@ethersproject/providers'
+import { BigNumber } from 'ethers'
+import { formatBigNumber } from '../../../src/util/NumberUtils'
+
+async function getInitialETHBalance(rpcURL: string): Promise<BigNumber> {
+  const goerliProvider = new JsonRpcProvider(rpcURL)
+  return await goerliProvider.getBalance(Cypress.env('ADDRESS'))
+}
+
+const goerliRPC = `https://goerli.infura.io/v3/${Cypress.env('INFURA_KEY')}`
+const arbitrumGoerliRPC = 'https://goerli-rollup.arbitrum.io/rpc'
+
+/**
+ * When user wants to bridge ETH from L1 to L2
+ */
+describe('Deposit ETH', () => {
+  // Happy Path
+  context('user has some ETH and is on L1', () => {
+    let l1ETHbal
+    let l2ETHbal
+
+    before(() => {
+      getInitialETHBalance(goerliRPC).then(
+        val => (l1ETHbal = formatBigNumber(val, 18, 5))
+      )
+      getInitialETHBalance(arbitrumGoerliRPC).then(
+        val => (l2ETHbal = formatBigNumber(val, 18, 5))
+      )
+      cy.disconnectMetamaskWalletFromAllDapps()
+      cy.login()
+    })
+
+    it('should show l2 chain id on url query param', () => {
+      // hardcoded to 421613 because e2e test is run on goerli
+      // TODO => add more tests for other chain ids?
+      cy.url().should('contain', '?l2ChainId=421613')
+    })
```

Comment:
This is cool, but we can remove it. 

We'll probably be changing these to `?from=1&to=42161` or something similar soon. The change does nothing for the user, but it would break the test, which we don't want.

Reply:
Instead, we can assert that the user sees `From: Goerli` and `To: Arbitrum Goerli`. 🤔

Reply:
+1. IMO testing the output (what the user sees) should be done rather than testing the implementation (url parameter)

Reply:
I think there's value in checking the URL as users sometimes interact with the browser's url bar but it isn't a must-have check